ignite/pkg/cosmosgen/generate_typescript.go

```diff
@@ -59,6 +59,11 @@ func (g *generator) generateTS(ctx context.Context) error {
 		return err
 	}
 
+	// add third party modules to for the root template.
+	for _, modules := range g.thirdModules {
+		data.Modules = append(data.Modules, modules...)
+	}
+
```

Comment:
same needs to be done on generate openapi

doesnt affect ts-client generation but it will be nice to polish everything

Reply:
sure thing!